2023-06-28T01:40:04.6435903Z ##[group]Run actions/setup-python@v4
2023-06-28T01:40:04.6436309Z with:
2023-06-28T01:40:04.6436551Z   python-version: 3.8
2023-06-28T01:40:04.6436863Z   architecture: x64
2023-06-28T01:40:04.6437158Z   cache: pip
2023-06-28T01:40:04.6437473Z   check-latest: false
2023-06-28T01:40:04.6437988Z   token: ***
2023-06-28T01:40:04.6438245Z   update-environment: true
2023-06-28T01:40:04.6438579Z   allow-prereleases: false
2023-06-28T01:40:04.6438920Z ##[endgroup]
2023-06-28T01:40:04.8082475Z ##[group]Installed versions
2023-06-28T01:40:04.8220797Z Successfully set up CPython (3.8.17)
2023-06-28T01:40:04.8224661Z ##[endgroup]
2023-06-28T01:40:05.5634744Z [command]/opt/hostedtoolcache/Python/3.8.17/x64/bin/pip cache dir
2023-06-28T01:40:06.0308927Z /home/runner/.cache/pip
2023-06-28T01:40:06.5072598Z Received 6339595 of 6339595 (100.0%), 37.1 MBs/sec
2023-06-28T01:40:06.5073506Z Cache Size: ~6 MB (6339595 B)
2023-06-28T01:40:06.5102367Z [command]/usr/bin/tar -xf /home/runner/work/_temp/729e6ec2-2fa5-4621-b1c9-fe7d76a045fc/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-06-28T01:40:06.5335001Z Cache restored successfully
2023-06-28T01:40:06.5354489Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.17-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
